monero-outreach-docs/translations/fr/website-sections/your_privacy_fr.md

```diff
@@ -0,0 +1,25 @@
+# Votre Confidentialité
+
+**"L'oiseau en cage chante d'un chant effrayé, de choses inconnues mais toujours désirées, et son chant est perçu sur les collines lointaines, car l'oiseau en cage chante à propos de la liberté" - Maya Angelou - 4th of January, 2021**
+
+Avec plusieurs politiques de confidentialité écrites dans le but de confondre et décourager la lecture, nous sommes reconnaissants d'avoir l'opportunité d'aborder votre vie privée, et notre responsabilité envers elle. Il s'agit de Monero, après tout, et on se doit de faire nous-mêmes ce que nous prêchons.
+
+TLDR: Notre site conserve le moins de données possibles, pour le moins longtemps possible, de la manière la plus sécure possible. Nous ne partagerons pas vos données avec qui que ce soit pour du profit, mais si ça se retrouve sur un serveur, considérez-le du domaine publique.
```

Comment:
de la manière la plus sécure  >>> de la manière la plus sûr 

Reason: Angliscisme

Reply:
Integrated, but it's "sûre" :)

Reply:
Yes. You are right :)